tests/integration/components/get-started-button/component-test.ts

```diff
@@ -0,0 +1,24 @@
+import { render } from '@ember/test-helpers';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupRenderingTest } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { module, test } from 'qunit';
+
+module('Integration | Component | Get Started button', hooks => {
+    setupRenderingTest(hooks);
+    setupMirage(hooks);
+
+    test('it renders', async function(assert) {
+        // Setup router
+        const router = this.owner.lookup('router:main');
+        router.setupRouter();
```

Comment:
Does this need the router?

Reply:
This test in particular needs the router because the button goes to the register route.  Without the router here, it times out and isn't able to render.